### DISCUSSION TITLE
[Enhancement] Upload Demo to Microsoft Windows Store，Google Play Store and Apple Store etc.

### DISCUSSION THREAD

**Discussion by jingliancui** [xamarin/XamarinCommunityToolkit#241] [Q&A]
## Summary
Could you consider upload the app to the stores？

**Comment by sthewissen** [comment]
I personally wonder if that would even be possible. It's a sample app, nothing more, nothing less. We would need to get all the legal bits in place, payments for different store accounts and all of the additional things that come with deploying to stores. Furthermore, I have my doubts that someone like Apple would even approve an app that's merely there for sample purposes.

What would be the use case for it being in the store? It's a developer tool primarily, so a developer can take it and run it on a device locally without too much effort.

**Comment by jingliancui** [comment]
You are right , my purpose is just want quickly see the feature ,may be you can uploade the gif to Microsoft docs and Github .md file (but github is very slow in china to load image)

**Comment by jfversluis** [comment]
@jingliancui thanks for your suggestion! Although I like the idea of having some kind of showcase app that makes this easier, I don't think it will happen soon. We need a lot of infrastructure (accounts, payment, etc.) like Steven mentioned for this.

What could make this easier for you? We are working on documentation on the official Microsoft Docs, if that is complete and has some screenshots, might that be enough for you?

**Comment by jingliancui** [comment]
@jfversluis Thansk, official Microsoft Docs is enough, maybe more gif images will be greate. And mention that image on github is very slow in China and sometime it doesn't load .

**Comment by powerdude** [comment]
I think it would be a great idea.  Look at the Syncfusion app at least in Google Play.  It showcases their controls and has links to github source.  I use it often to get ideas or as a refresher on functionality.

**Comment by AlexHedley** [comment]
I can see this being a lot of hurdles to jump through.
It could be added to an App Center deployment lifecycle to automate.

Example Apps:

Telerik UI for Xamarin Samples
https://apps.apple.com/gb/app/telerik-ui-for-xamarin-samples/id1083924868

Where to find Syncfusion Xamarin Forms Sample Apps
https://www.syncfusion.com/kb/10811/where-to-find-syncfusion-xamarin-forms-sample-apps
There was an iOS app but it might have been pulled now.

**Comment by pictos** [comment]
@jfversluis  We can use the AppCenter for this, the Essentials uses this [approach](https://github.com/xamarin/essentials#sample-app).

**Reply by jfversluis** [reply]
Well, the last version is from May. So I don't think they're actually really doing it right now anymore :D

**Reply by domagojmedo** [reply]
I set up Android through AppCenter in minutes and it all worked great. Making it auto build and distribute through AppCenter on pushes to main seems like it shouldn't be a problem. iOS is a different story, but I think that shouldn't stop Android sample app from being available

https://install.appcenter.ms/users/medo.moberg/apps/xct/distribution_groups/public
